packages/pluggableWidgets/datagrid-web/src/components/Header.tsx

```diff
@@ -130,6 +142,7 @@ function useDraggable<D extends object>(
 
     const handleOnDrop = useCallback(
         (e: DragEvent<HTMLDivElement>): void => {
+            setIsDragging(false);
```

Comment:
Optional comment: 
How would it look if you just call handleDragEnd() and remove line 145 and 146.

Reply:
Done, good one!!